docs/cuproj/source/user_guide/cuproj_api_examples.ipynb

```diff
@@ -0,0 +1,192 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c5fdf490-fa77-4e56-92d1-53101fff75ba",
+   "metadata": {},
+   "source": [
+    "# cuProj Python User's Guide\n",
+    "\n",
+    "cuProj is a GPU-accelerated Python library for cartographic coordinate projection and coordinate transformations between coordinate reference systems (CRS). The cuProj Python API provides an accessible interface to high-performance projections accelerated by CUDA-enabled GPUs. The API closely follows the [PyProj](https://pyproj4.github.io/pyproj/stable/) API."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "caadf3ca-be3c-4523-877c-4c35dd25093a",
+   "metadata": {},
+   "source": [
+    "## Contents\n",
+    "\n",
+    "This guide provides a working example for all of the python API components of cuProj.  \n",
+    "The following list links to each subsection.\n",
+    "\n",
+    "* [Installing cuProj](#Installing-cuProj)\n",
+    "* [Transformations with Transformer](#Transformations-with-Transformer)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "115c8382-f83f-476f-9a26-a64a45b3a8da",
+   "metadata": {},
+   "source": [
+    "## Installing cuProj\n",
+    "Read the [RAPIDS Quickstart Guide](  https://rapids.ai/start.html     ) to learn more about installing all RAPIDS libraries, including cuProj.\n",
+    "\n",
+    "If you are working on a system with a CUDA-enabled GPU and have CUDA installed, uncomment the  \n",
+    "following cell and install cuSpatial:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "7265f9d2-9203-4da2-bbb2-b35c7f933641",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# !conda create -n rapids-23.08 -c rapidsai -c conda-forge -c nvidia \\ \n",
+    "#     cuproj=23.08 python=3.10 cudatoolkit=12.0"
```

Comment:
```suggestion
    "# !conda create -n rapids-23.08 --solver=libmamba -c rapidsai -c conda-forge -c nvidia \\ \n",
    "#     cuproj=23.08 python=3.10 cuda-version=12.0"
```

The `12.0` support work led to us swapping over to `cuda-version` from `cudatoolkit` because now you can define parts of the CTK you need instead of the kitchen sink. Also, we're default recommending the mamba solver since it's much, much faster and included in conda now.